kpet/run.py

```diff
@@ -260,8 +258,6 @@ def generate(self, description, kernel_location, lint, variables,
             TREE=tree_name,
             RECIPESETS=self.recipesets_of_hosts,
             VARIABLES=variables,
-            getenv=os.getenv,
```

Comment:
`getenv` is not needed, either?

Reply:
Yep, it was needed for just that one variable.